src/components/Inbox/index.tsx

```diff
@@ -180,11 +181,11 @@ const Checkbox = styled(MUICheckbox)`
   }
 `;
 
-const Container = styled(Grid)`
+const Box = styled(MUIBox)`
+  height: 90vh;
   overflow: auto;
-  height: 100vh;
-  max-height: 100vh;
-  width: 387.3px;
+  display: flex;
+  flex-direction: column;
```

Comment:
not a big deal but you can actually set a lot of these values through the props for `Box` directly:
`<Box height="90vh" display="flex" flexDirection="column" overflow="auto">...</Box>`